### ISSUE TITLE
Give Admin The Functionality To Delete And Edit Images Included In Blogs 

### ISSUE THREAD

**Issue by Indk0** [Indk0/FinMeBlog#27]
As a developer I need to give admin the ability to edit and delete blog images.

- [x] Create and code admin image and alt functionality.
- [x] Provide default image and alt text if no other image is uploaded or no alt text is provided.
- [x] Test and deploy.
